makefile

```diff
@@ -517,12 +557,17 @@ status info:
 	$(info MFEM_USE_SIDRE       = $(MFEM_USE_SIDRE))
 	$(info MFEM_USE_CONDUIT     = $(MFEM_USE_CONDUIT))
 	$(info MFEM_USE_PUMI        = $(MFEM_USE_PUMI))
+	$(info MFEM_USE_CUDA        = $(MFEM_USE_CUDA))
+	$(info MFEM_USE_RAJA        = $(MFEM_USE_RAJA))
+	$(info MFEM_USE_OCCA        = $(MFEM_USE_OCCA))
+	$(info MFEM_USE_MM          = $(MFEM_USE_MM))
```

Comment:
Document these new `MFEM_USE_*` variables in the `INSTALL` file. In particular document the install process for CUDA, RAJA and OCCA, and any expectations on the location of the installed directories.